src/Http/Controllers/SwaggerController.php

```diff
@@ -2,9 +2,10 @@
 
 namespace L5Swagger\Http\Controllers;
 
-use File;
-use Request;
-use Response;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Request;
+use Illuminate\Support\Facades\Response;
```

Comment:
Maybe we can use `\Illuminate\Http\Response` to rid of facades?

Reply:
I think we can't because there is no static `make` method.